packages/host/src/components/AppDrawer/AppDrawer.tsx

```diff
@@ -207,6 +209,11 @@ export const AppDrawer: React.FC = () => {
             icon={<RadioIcon fontSize="small" color="primary" />}
             text={t('sync')}
           />
+          <ExternalNavLink
+            to={ExternalURL.PublicDocs}
+            icon={<BookIcon fontSize="small" color="primary" />}
```

Comment:
thinking that if/when we do have specific docs links for specific pages then we can manage that in a number of different ways. That was part of the reason for having the docs link in the AppBarButtons - but I like the button better tucked away in the left nav

Reply:
Yup!